src/test/xspec/dtbook-to-epub3.xspec

```diff
@@ -820,13 +820,34 @@
     <!-- implicit epub:type tested in element test, otherwise nothing interesting to test -->
 
     <!-- epigraph -->
+    <!-- The nordic guidelines do not allow for epigraph. So the
+         following is for completeness sake and takes some inspiration
+         from https://www.w3.org/TR/dpub-aria-1.0/#doc-epigraph -->
     <x:scenario label="when processing a epigraph element">
-        <x:context>
-            <dtbook:epigraph/>
-        </x:context>
-        <x:expect label="the resulting element should be a p with an added epub:type of epigraph">
-            <html:p id="..." epub:type="epigraph" class="epigraph"/>
-        </x:expect>
+	<x:scenario label="on its own">
+            <x:context>
+		<dtbook:epigraph/>
+            </x:context>
+            <x:expect label="the resulting element should be a blockquote with an added epub:type of epigraph">
+		<html:blockquote id="..." epub:type="epigraph" class="epigraph"/>
+            </x:expect>
+	</x:scenario>
+	<x:scenario label="which is inside a p">
+            <x:context>
+		<dtbook:p>foo<dtbook:epigraph/>bar</dtbook:p>
+            </x:context>
+            <x:expect label="the resulting element should be a p with a blockquote inside">
+	        <html:p>foo<html:blockquote id="..." epub:type="epigraph" class="epigraph"/>bar</html:p>
```

Comment:
According to `nordic-html5.rng`, `<p>` may only contain inline elements, and `<blockquote>` is a block element.

Reply:
Yes, agreed. Was a left over test from the previous PR